_includes/docs/latest/ajax-api.html

```diff
@@ -1045,8 +1045,8 @@ <h3 id="api-schedule-a-flow">Schedule a Flow</h3>
       <td>The name of the flow.</td>
     </tr>
     <tr>
-      <td>scheduleTime</td>
-      <td>The time to schedule the flow. Example: 12,00,pm,PDT</td>
+      <td>scheduleTime(with timezone)</td>
+      <td>The time to schedule the flow. Example: 12,00,pm,PDT (Unless UTC is specified, Azkaban will take current server's default timezone instead)</td>
```

Comment:
Do you mean the specified time zone at the end will be ignored unless it is UTC?